pandas/tests/series/test_arithmetic.py

```diff
@@ -131,6 +286,50 @@ def test_comparison_flex_basic(self):
             with pytest.raises(ValueError, match=msg):
                 getattr(left, op)(right, axis=1)
 
+    def test_comparison_flex_alignment(self):
```

Comment:
I thought you were splitting out arithemtic & comparisons elsewhere? seems reasonable to do here as well.

Reply:
this is in the class `TestSeriesFlexComparison`